spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,6 +32,59 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile5.0-RC1]]
+=== MicroProfile 5.0 Release Candidate 1
+
+MicroProfile 5.0-RC1 aligns with Jakarta EE 9.1. It means this release will not work with previous version of Jakarta EE such as Jakarta EE 8. This effectively creates an incompatible change due to the namespace change from javax to jakarta in the code. This release has no other functional update when compared to MicroProfile Release 4.1, except the dependencies on Jakarta namespaces.
+
+    Config 3.0
+    Fault Tolerance 4.0
+    Health 4.0
+    JWT RBAC 2.0
+    Metrics 4.0
+    Open API 3.0
+    Open Tracing 3.0
+    Rest Client 3.0
+
+The following specifications include API incompatible changes:
+
+* https://download.eclipse.org/microprofile/staging/microprofile-config-3.0-RC5/microprofile-config-spec-3.0-RC5.html#_incompatible_changes[Config]
+* https://download.eclipse.org/microprofile/staging/microprofile-fault-tolerance-4.0-RC2/microprofile-fault-tolerance-spec-4.0-RC2.html#_incompatible_changes[Fault Tolerance]
```

Comment:
these should be moved out of /staging directory before we push 5.0-RC1

Reply:
@emecas let's monitor these staging links. When the links without the "staging" part is available, the links can be updated here

Reply:
I'll release more update for the specs in the next couple of days and will push the latest out of staging.